### ISSUE TITLE
Incorrect character position in error messages

### ISSUE THREAD

**Issue by vaitkus** [merkys/Chemistry-OpenSMILES#4]
For example, parsing SMILES `C-	1000000` outputs the following warning:
```
./smi_parse: <FILE>(1) 1000000: syntax error at position 24.
```
Since the file only contains 10 visible characters, position 24 seems quite strange.

**Comment by merkys** [comment]
Fixed in 76c5085a9b5507e7a1e70ff6e2c2a1ed9d1a0789. Thanks for reporting.

**Comment by vaitkus** [comment]
Great, thank you.
